lib/src/main/java/io/ably/lib/http/Http.java

```diff
@@ -404,6 +395,19 @@ private URL buildURL(String uri, Param[] params) {
 		return result;
 	}
 
+	/**
+	 * TODO: Fill javadoc
```

Comment:
If this is private we don't necessarily need a javadoc comment, but please either add a comment or remove the TODO. It's not worth leaving like this.